uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
```

Comment:
usually BAO classes are created if there is an entity supporting it , so for example in job contract  extension you have HRJobDetails entity which have a DAO (which mean this is an entity which have a table in the DB) and BAO  classes . so I think no need.